jib-core/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTest.java

```diff
@@ -351,7 +348,7 @@ public void setApplyContainerBuildPlan()
     Assert.assertEquals(Arrays.asList("bar", "cmd"), containerConfiguration.getProgramArguments());
 
     ContainerBuildPlan convertedPlan = containerBuilder.toContainerBuildPlan();
-    Assert.assertEquals("arch", convertedPlan.getArchitectureHint());
-    Assert.assertEquals("os", convertedPlan.getOsHint());
+    Assert.assertEquals(
+        ImmutableSet.of(new Platform("testOS", "testArchitecture")), convertedPlan.getPlatforms());
```

Comment:
```suggestion
        ImmutableSet.of(new Platform("testArchitecture", "testOs")), convertedPlan.getPlatforms());
```

Reply:
Did we already update the platform api to change the argument order? The one we deployed.

Reply:
Yeah. If you look into the jib-build-plan 3.0 library, you'll see the actual code.

![image](https://user-images.githubusercontent.com/10523105/87802001-89852580-c81e-11ea-8d71-6c89534bfc63.png)

Reply:
Nice one.